ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -78,10 +78,14 @@ images:
         ostree container commit
   from: ocp_os
   to: custom-os-image
+- dockerfile_literal: |
+    FROM quay.io/zzlotnik/zacks-openshift-helpers:latest
```

Comment:
thought (non-blocking): Once https://github.com/openshift/machine-config-operator/pull/4718 lands, we'll need to tweak this job slightly to build the tools from the MCO repository instead of pulling them from this container image.

Reply:
makes sense